src/extension.ts

```diff
@@ -169,13 +169,14 @@ export function activate(context: vscode.ExtensionContext): void {
     });
     initAsyncCommand(context, 'diagnostics.OpenLogStream', async (node: SiteNodeBase) => {
         if (node) {
-            const enableButton = 'Yes';
+            const enableButton: vscode.MessageItem = { title: 'Yes' };
+            const notNowButton: vscode.MessageItem = { title: 'Not Now', isCloseAffordance: true };
             const isEnabled = await vscode.window.withProgress({ location: vscode.ProgressLocation.Window }, p => {
                 p.report({ message: 'Checking container diagnostics settings...' });
                 return node.isHttpLogsEnabled();
             });
 
-            if (!isEnabled && enableButton === await vscode.window.showWarningMessage('Do you want to enable logging and restart this container?', enableButton)) {
+            if (!isEnabled && enableButton === await vscode.window.showWarningMessage('Do you want to enable logging and restart this container?', enableButton, notNowButton)) {
```

Comment:
Usually we would give them 'Okay', 'Not Now', _and_ 'Cancel'

Reply:
I don't think this prompt is very clear as if you were to hit 'Cancel' today, it would still show a log-stream (what's available without enabling logging)

Reply:
I don't have a strong opinion on this - sounds fine to me